demo/admin/src/App.tsx

```diff
@@ -54,6 +55,10 @@ const apolloClient = createApolloClient(config.apiUrl);
 declare module "@comet/cms-admin" {
     // eslint-disable-next-line @typescript-eslint/no-empty-object-type
     interface ContentScope extends BaseContentScope {}
+
+    export interface PermissionOverrides {
+        app: GQLAppPermission;
```

Comment:
It's odd that we need this additional "app" level here, but that's how module augmentation works.